### ISSUE TITLE
'props is not defined' when using Playground

### ISSUE THREAD

**Issue by zlw11063** [pedronauck/docz#475]
## Bug Report

**Describe the bug**

When I use the Playground component, I'm receiving this error:

```props is not defined```

My guess is that it is probably something I've done wrong. I do not have this issue in other repositories where I've begun using docz. This repository is relatively simple - nothing special about it. It only contains a single React component. It was built with Create-React-App (react-scripts version at the bottom). 

Removing and reinstalling node modules didn't affect it.

I've created a very simple card-style component that wraps around the docz Link component. It does nothing else. Even just using that causes the error. I have a few other components I've made I'd like to use, but this issue appears when I use any component in the Playground. Here is some example code that produces the error:

```js
---
name: Simple Component
menu: Examples
route: /simple-component
---

import CardLink from './CardLink.jsx';
import { Playground } from 'docz';

# Simple Component

<Playground>
  <CardLink to="/simple-component" title="Example" />
</Playground>

```

Stack Trace as provided by docz on the crash page:

```
in MDXContent (created by AsyncComponent)
in AsyncComponent (created by Component)
in Component (created by t)
in t (created by y)
in y (created by c)
in c (created by ForwardRef)
in Component (created by AsyncRoute)
in div (created by Styled(div))
in Styled(div) (created by Context.Consumer)
in div (created by Styled(div))
in Styled(div) (created by Context.Consumer)
in div (created by Styled(div))
in Styled(div) (created by St)
in St (created by Context.Consumer)
in ThemeConfig (created by Gt)
in Gt (created by AsyncRoute)
in AsyncRoute (created by t)
in t (created by Context.Consumer)
in t (created by Context.Consumer)
in MDXProvider (created by DocPreview)
in DocPreview (created by Context.Consumer)
in ThemeProvider (created by Context.Consumer)
in ThemeConfig (created by Component)
in Component (created by Component)
in DefaultWrapper (created by Component)
in ScrollToTopBase (created by t)
in t (created by withRouter(ScrollToTopBase))
in withRouter(ScrollToTopBase) (created by Component)
in t (created by t)
in t (created by Component)
in DataServer (created by Component)
in Provider (created by Component)
in ErrorBoundary (created by Component)
in Component (at root.jsx:6)
in Root (created by HotExportedRoot)
in AppContainer (created by HotExportedRoot)
in HotExportedRoot (at app/index.jsx:15)
```

When looking at the code in the dev console in Chrome, this is where it shows the error occurring:

```js
import React from 'react'
import { MDXTag } from '@mdx-js/tag'
import CardLink from './CardLink.jsx';
import { Playground } from 'docz';

export default class MDXContent extends React.Component {
  constructor(props) {
    super(props)
    this.layout = undefined
  }
  render() {
    const { components = {} } = this.props

    return <MDXTag name="wrapper" components={components}>
                    <MDXTag name="h1" components={components} props={{"id":"simple-component"}}> 
                          {`Simple Component`}
                    </MDXTag>
                    <Playground __codesandbox={`bigfatcodesandboxmagicwashere`} __position={0} __code={`<CardLink to="/simple-component" title="Example" />`} __scope={{props,CardLink}}>
                        <CardLink to="/simple-component" title="Example" />
                    </Playground>
           </MDXTag>
  }
}
```

It looks like the `props` parameter in `__scope` on the playground isn't defined.

**Environment**

* OS: OSX 10.14.1 
* Node/npm version: Node v8.11.4 / NPM v5.6.0
* React: 16.6.3

**Additional context/Screenshots**
I am working with several native web components wrapped in a react component, though the issue still occurs when they aren't included.

**Dependencies**
```json
"dependencies": {
    "react": "^16.6.3",
    "react-dom": "^16.6.3",
    "react-scripts": "2.1.1"
  },
"devDependencies": {
    "docz": "^0.12.12",
    "docz-plugin-css": "^0.11.0"
  }
```

**doczrc.js**
```js
// doczrc.js
import {css} from 'docz-plugin-css';

export default {
  plugins: [
    css()
  ]
}
```

Again, I'm sure it's something I did. I'm only having this issue in this repository - so I figure its something you good people probably would know how to fix.

Great work on this whole project. Docz makes documentation a dream.

**Comment by zWingz** [comment]
`release@0.12.13` fixed it #463

**Comment by zlw11063** [comment]
Updating fixed this issue. Thanks!
